modules/http-load-balancer-website/variables.tf

```diff
@@ -119,31 +136,37 @@ variable "access_log_prefix" {
 # https://github.com/terraform-providers/terraform-provider-google/issues/3134
 #variable "website_kms_key_name" {
 #  description = "A Cloud KMS key that will be used to encrypt objects inserted into the website bucket. If empty, the contents will not be encrypted. You must pay attention to whether the crypto key is available in the location that this bucket is created in."
+#  type = string
 #  default     = ""
 #}
 
 #variable "access_logs_kms_key_name" {
 #  description = "A Cloud KMS key that will be used to encrypt objects inserted into the access logs bucket. If empty, the contents will not be encrypted. You must pay attention to whether the crypto key is available in the location that this bucket is created in."
+#  type = string
 #  default     = ""
 #}
```

Comment:
Did you intend to fix these as well, since you enabled them in the other module?

Reply:
Good catch!